packages/frontend/core/src/components/pure/workspace-mode-filter-tab/index.css.ts

```diff
@@ -0,0 +1,9 @@
+import { style } from '@vanilla-extract/css';
+
+export const filterTab = style({
+  padding: '4px 8px',
```

Comment:
The generic `RadioGroup` should have same style, why redefine `padding` here?

Reply:
Oh, I forgot to remove the definition of this attribute.